tests/_utils/wait_process_exit

```diff
@@ -2,37 +2,21 @@
 # parameter 1: process name
 process=$1
 
-# Number of time to wait before using "kill -9"
-WAIT_COUNT=3
-
-# Counter to keep count of how many seconds have passed
-count=0
+while true; do
+	pgrep $process >/dev/null 2>&1
+	ret=$?
+	if [ "$ret" != "0" ]; then
+		echo "process $process already exit"
+		break
+	fi
+	sleep 0.2
+	echo "wait process $process exit..."
+done
```

Comment:
seems we don't need lines after line 15

Reply:
i not sure but i found in some case , process still running after `wait_process_exit`,so i add `kill -9` to guarantee it

Reply:
when `while true` break, line 17 is always true 🤔

Reply:
🙄  you are right, move `kill -9` inside loop

Reply:
after kill -9, still need to wait processs exit